test/testgroups

```diff
@@ -5,4 +5,6 @@ dynamic_system
 circuit
 gate
 transpile
-submit_circuit
\ No newline at end of file
+submit_circuit
```

Comment:
Wasn't `submit_circuit` removed?

Reply:
only the call to `push_gate!(c, [sigma_x(1)])` was commented, as the push cannot yet handle AbstractGates